modules/ROOT/pages/dw-core-functions-filter.adoc

```diff
@@ -146,6 +146,78 @@ users filter ((item, index) -> item.error == null)
 ----
 
 
+=== Example
```

Comment:
TODO (DEV): OK with adding this example? It was in v4.3 docs but not DWL.

TODO (DUKE): add to Core.dwl if approved.